app/src/main/res/values-om-rET/strings.xml

```diff
@@ -254,6 +259,15 @@
 
   <!-- Onboarding -->
   <string name="onboarding_get_started">Jalqabi</string>
+  <string name="screenonboarding_intro_1">Barbaadi \&amp; argadhu \ndhukkubsattoota kumaatamoota \nwajjiin</string>
+  <string name="screenonboarding_intro_1_hypertension">Baayyina dhiiga</string>
+  <string name="screenonboarding_intro_2">Galmeessa \n tursiisi</string>
+  <string name="screenonboarding_intro_2_bp">Dhiibbaa dhiigaa</string>
```

Comment:
Can you add `\ ` front slash and space just like we have in the `strings.xml`. The space between text in the UI is messed up because of this.